content/change/teaching/resources.md

```diff
@@ -1,11 +1,12 @@
-title: Resources
+title: Teaching resources
 
 # Where can I find resources for teaching open science?
 
 Open Science is here to stay. Therefore, students deserve to be prepared for a future where reproducable research practices are standard. We have found that students intuitively understand the necessity for changes to the scientific practice, and enjoy applying state-of-the-art methods in their projects.
 
 As with scientific resources, members of the Open Science community have created and share freely their teaching materials. Here are a few resources:
 
+* The [**Open Science Training Handbook**](https://book.fosteropenscience.eu) is a fantastic and comprehensive resource that covers many topics around open science. It's built for people who would like to create open science training, and very in-depth.
```

Comment:
This is great! I've been looking for a place to link to them, this fits perfectly. Thanks!